make.js

```diff
@@ -20,7 +20,7 @@
 
 'use strict';
 
-require('./external/shelljs/make');
+require('shelljs/make');
```

Comment:
I'm wondering if it would be a good idea to enclose this in a `try... catch` block, e.g. something like this:
```javascript
try {
  require('shelljs/make');
} catch (ex) {
  console.log('Run "npm install"'); // Probably needs a better message
  return;
}
```
When I tested the patch the first time, running any of the `make` commands just threw an error, and it took me a couple of seconds to realise that I needed to run `npm install`.
For less experienced users, it's not unlikely that they *could* get stuck on this, so a message telling them what to do seems like a good idea to me.

*Edit:* A somewhat fun fact is that if the above is implemented, I think that it would actually make PR #5908 obsolete :-)